adapters/terraform/aws/s3/bucket.go

```diff
@@ -74,13 +171,24 @@ func hasLogging(b *terraform.Block) types.BoolValue {
 		}
 		return types.BoolDefault(false, loggingBlock.GetMetadata())
 	}
+	if targetBucket := b.GetAttribute("target_bucket"); targetBucket.IsNotNil() {
+		return types.Bool(true, b.GetMetadata())
+	}
 	return types.BoolDefault(false, b.GetMetadata())
 }
 
 func isVersioned(b *terraform.Block) types.BoolValue {
 	if versioningBlock := b.GetBlock("versioning"); versioningBlock.IsNotNil() {
 		return versioningBlock.GetAttribute("enabled").AsBoolValueOrDefault(true, versioningBlock)
 	}
+	if versioningBlock := b.GetBlock("versioning_configuration"); versioningBlock.IsNotNil() {
+		status := versioningBlock.GetAttribute("status")
+		if status.Equals("Enabled", terraform.IgnoreCase) {
+			return types.Bool(true, versioningBlock.GetMetadata())
+		} else {
+			return types.Bool(false, versioningBlock.GetMetadata())
```

Comment:
The block will be `nil` here so we need the parent blocks metadata.
```suggestion
			return types.Bool(false, b.GetMetadata())
```